### PR TITLE
chore(docs): some boilerplate updates with the new options

### PR DESCRIPTION
**Pull request by zebateira**

### CODE REVIEW

**Comment by terichadbourne** [comment]
@zebateira - I pushed a commit with some suggested edits to give a bit more context. Let me know what you think! 

There's currently no detailed documentation for the `modules` option, so I think we should add that before merging this. Some lessons, like in Basics, use an import statement to access CID, and a bunch of other lessons say they're using the CID module via the option but don't actually seem to use it.

**Comment by zebateira** [comment]
@terichadbourne updated 👍

**Comment by terichadbourne** [comment]
@zebateira I tweaked your formatting in a way that would make more sense to me as a reader - let me know if this still works.

File: src/tutorials/boilerplates/boilerplate-code.js [hunk 1]
```diff
@@ -31,8 +31,8 @@ return run
 `
 
 // Optional: Use the modules option if you need extra
-// modules available in the solution code
-const modules = { cids: require('cids') }
+// modules available in the validation code
```
Comment (zebateira) [inline src/tutorials/boilerplates/boilerplate-code.js]:
```suggestion
// modules available in the solution code
```

Comment (zebateira) [inline src/tutorials/boilerplates/boilerplate-code.js]: @terichadbourne  Should be solution code, not validation. Basically modules available to the user when writing up the solution code.

```suggestion
// modules available in the solution code
```

**Comment by zebateira** [comment]
@terichadbourne all good. just one last comment

**Pull request merged**